_includes/tutorials/message-ordering/kafka/code/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.21.0"
+        classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+    }
+}
+
+plugins {
+    id "java"
+    id "com.google.cloud.tools.jib" version "1.1.1"
```

Comment:
Is this needed?

Reply:
@mikebin this line is removed in https://github.com/confluentinc/kafka-tutorials/pull/587/commits/12e0b49c39a1035b172e96a1c1835347b5900090